libs/application/templates/children-domicile-transfer/src/fields/ChangeInformation/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { Box, Text } from '@island.is/island-ui/core'
+import { extractParentFromApplication } from '../../lib/utils'
+import { CheckboxController } from '@island.is/shared/form-fields'
+
+const ChangeInformation = ({ field, application, error }: FieldBaseProps) => {
+  const { id, disabled } = field
+  const parent = extractParentFromApplication(application)
+  const parentAddress = `${parent?.address}, ${parent?.postalCode} ${parent?.city}`
```

Comment:
I was wondering with this line do we want to extract the formatting of the string into a helper function? 

I am guessing we are always going to show the address in this format so it could be nice to have it in one place if we decide to change the format.

Reply:
Done 💯